src/features/defi/components/PairIcons/PairIcons.tsx

```diff
@@ -90,11 +93,12 @@ export const PairIcons = ({
       />
     ))
   }, [iconBoxSize, iconSize, icons, showFirst])
+
   return (
-    <Flex display='inline-flex' flexDirection='row' alignItems='center' {...styleProps}>
-      {showFirst && <Avatar src={firstIcon} size={iconSize} boxSize={iconBoxSize} />}
+    <Box display='inline-flex' flexDirection='row' alignItems='center' {...styleProps}>
```

Comment:
q: Do we need this?

Reply:
Reverted in [fc30d22](https://github.com/shapeshift/web/pull/7492/commits/fc30d22890df089e3ab57cd4c8c152afb8e47eaa)